routes/jwtauth/jwtRefresh.js

```diff
@@ -0,0 +1,71 @@
+import { verify as _verify } from "jsonwebtoken";
+import moment from "moment";
+import { getClientIp } from "request-ip";
+import jwtVerify from "./jwtVerify";
+import jwtBlock from "./jwtBlock";
+import { getNewSignedAccessJWTToken as jwtSign } from "./jwtSign";
+import authLog from "../../models/authlog";
+import User from "../../models/user";
+  
+const jwtRefresh = async (req, accesstoken, refreshtoken) => {
```

Comment:
Codacy found an issue: [Async arrow function has a complexity of 8. Maximum allowed is 4.](https://app.codacy.com/gh/implude/local-community-backend/pullRequest?prid=6225533)